README.md

```diff
@@ -1,3 +1,3 @@
-# SalesforceDayTwo
+# Salesforce Day Two
 
-Welcome to your second day of Git learning!
\ No newline at end of file
+Welcome to OUR second morning of Gittastic learning!
```

Comment:
This line is a good change!

Reply:
Here's how I would've done this feature / an example of what I'm thinking

```javascript
console.log("Hello World");
for(var i of array) {
  console.log(i)
}
```

Reply:
test two
```suggestion
Welcome to OUR 2nd morning of Gittastic learning!
```